src/lib/components/FrontPage.svelte

```diff
@@ -9,14 +12,25 @@
 		if (inputfield.length > 6) {
 			inputfield = inputfield.slice(0, -1);
 		}
+
+    if(boardExists() === true){
+      color = "green-500"
+
+    }else{
+      color = "red-500"
+    }
```

Comment:
Hej Thorbjørn,

Har du overvejet følgende?
```suggestion
    color = boardExists() === true ? "green-500" : "red-500";
```